examples/fluids/problems/newtonian.c

```diff
@@ -202,6 +204,11 @@ PetscErrorCode NS_NEWTONIAN_IG(ProblemData *problem, DM dm, void *ctx, SimpleBC
   if (state_var == STATEVAR_PRIMITIVE && !implicit) {
     SETERRQ(comm, PETSC_ERR_ARG_NULL, "RHSFunction is not provided for primitive variables (use -state_var primitive only with -implicit)\n");
   }
+
+  PetscCall(PetscOptionsScalar("-idl_decay_time", "Effectively inverse of strength of IDL. The timestep is good starting point", NULL, idl_decay_time,
```

Comment:
Similarly here, "Effectively inverse of strength of IDL" should be replaced by something more informative/intuitive.

Reply:
Let $p = p_{ref} + p'$. The deviation will decay like $p'(t) = e^{-t/\tau} p'(0)$ where $\tau$ is the characteristic decay time. It's similar to a "half life" of the pressure anomalies.